WeakableSelf.podspec

```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
   s.name             = 'WeakableSelf'
   s.version          = '1.1.0'
-  s.summary          = 'KeyPathKit leverages Swift 4 KeyPath type in order to implement a SQL-like data manipulation API'
+  s.summary          = 'A Swift micro-framework to easily deal with weak references to self inside closures'
```

Comment:
Good catch! That's were it shows that I published this micro-framework overnight 😂